libmisc/salt.c

```diff
@@ -263,6 +267,78 @@ static /*@observer@*/const char *gensalt_bcrypt (void)
 }
 #endif /* USE_BCRYPT */
 
+#ifdef USE_YESCRYPT
+/* Default cost if not explicitly specified.  */
+#define Y_COST_DEFAULT 8
```

Comment:
BTW, libxcrypt's default is 5, not 8. With 8, it's 128 MiB and ~100 ms - probably generally affordable these days when not under attack, but probably does make (inadvertent) DoS of some systems/services easier.

(This is not an objection. Just a comment.)

Reply:
You are right, 5 seams much more reasonable.